tgui/packages/tgui/interfaces/ConstructionForklift/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { useBackend } from '../../backend';
+import { Button, Image, Section } from '../../components';
+import { Window } from '../../layouts';
+
+export const ConstructionForklift = (props: any) => {
+  const { act, data } = useBackend<ForkliftData>();
+  const { materials, modules, cooldowns, hologram_count, active_module_data } =
+    data;
+
+  let build_select: JSX.Element | undefined;
+  if (active_module_data !== undefined) {
+    build_select = (
+      <BuildTargetSelect
+        selected={active_module_data.current_selected_typepath}
+        available_builds={active_module_data.available_builds}
+      />
+    );
+  }
+
+  const set_active_module = (module: string) => {
+    act('set-active', { new_module_ref: module });
+  };
+  let module_select = (
+    <ModuleSelect modules={modules} set_active_module={set_active_module} />
```

Comment:
just put these in the jsx please, no need to make them a var outside of it,

```tsx
<Window.Content>
{!!active_module_data && (<BuildTargetSelect
        selected={active_module_data.current_selected_typepath}
        available_builds={active_module_data.available_builds}
      />)}
<ModuleSelect modules={modules} set_active_module={() =>  act('set-active', { new_module_ref: module });} />
 </Window.Content>
 ```

Reply:
I prefer it like this as it's cleaner and imo more legible